functions/signup-processor/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM public.ecr.aws/lambda/nodejs:22 AS builder
+WORKDIR /usr/app
+COPY package.json pnpm-lock.yaml tsconfig.json index.ts db.ts ./
+RUN npm install -g pnpm
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)